pages/_app.js

```diff
@@ -1,7 +1,15 @@
+import { ThemeProvider } from 'styled-components';
 import '../styles/globals.css';
+import { createGlobalStyle } from 'styled-components';
 
 function MyApp({ Component, pageProps }) {
   return <Component {...pageProps} />;
 }
 
 export default MyApp;
+
+const GlobalStyle = createGlobalStyle`
+  body {
+    margin: 0;
+    padding: 0;
+  }`;
```

Comment:
You don't use the GlobalStyle Component in `MyApp`:
```suggestion
function MyApp({ Component, pageProps }) {
  return (<>
    <GlobalStyle />
    <Component {...pageProps} />;
  </>)
}


export default MyApp;

const GlobalStyle = createGlobalStyle`
  body {
    margin: 0;
    padding: 0;
  }`;
```